Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM ubuntu
+FROM alpine
 
-ADD ipam-driver /
+RUN apk update
```

Comment:
You're running `apk update` but never `add`. So it's just adding bulk to the image. More often you want to do the update, the add, and then delete the cache files. Like this: https://github.com/infobloxopen/dnstools/blob/master/Dockerfile#L4